### DISCUSSION TITLE
multiple postgres host ips in kong.conf

### DISCUSSION THREAD

**Discussion by err-himanshu** [Kong/kong#7274] [Ideas and feature requests]
Hello
How to provide multiple postgres ips in kong.conf , I have tried ips with space seprated but it seems it only takes the first ip
like
pg_host = xx.xxx.xx.xxx xx.xxx.xx.xxx
                    **OR**
pg_host = xx.xxx.xx.xxx,xx.xxx.xx.xxx

PS: Comma separated ips ain't working.

**Comment by bungle** [comment]
@err-himanshu,

Yes that is not supported at the moment. But we have noted that. Right now you can perhaps use a proxy in front of postgres servers.

**Reply by liukaiyuliukaiyu** [reply]
Are there any plans to support this feature multiple postgres host? If I use a proxy  in front of postgres server，do I need to restart the gateway when the server instance changes? because the pg_host value is fixed.

**Comment by err-himanshu** [comment]
@bungle but in kong.conf we have to define the the pg_user and pg_password and it ain't working with proxy server 
getting error 
**could not retrieve server_version: connection refused**

**Comment by bungle** [comment]
@err-himanshu, you cannot connect your PG at all right? We need to know a way more from your setup, until we can help you. Is this another separate issue than the multiple hosts? How are you running your PG, can you connect Postgres with `psql` from the node where you run Kong? For me it looks like the host cannot be reached from Kong. Is there indirection between Kong and Posgres, and if there is, what kind of indirection?
